src/scripts/main.js

```diff
@@ -22,7 +22,23 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  element.innerHTML += `
+    <ul></ul>
+  `;
+
+  for (const key in data) {
+    const list = element.lastElementChild;
+
+    list.innerHTML += `
+      <li>${key}</li>
+    `;
+
+    const keys = Object.keys(data[key]);
+
+    if (keys) {
```

Comment:
an array is always a truth value
```suggestion
    if (keys.length) {
```